src/nomadic/realtime/commands.py

```diff
@@ -153,12 +154,9 @@ def realtime(
                 message=f"Region BED file not found at {region_bed}.",
             )
 
-    if fastq_dir is None:
-        fastq_dir = default_fastq_dir(experiment_name)
-        if not os.path.isdir(fastq_dir):
-            raise click.BadParameter(
-                message=f"FASTQ directory not found at {fastq_dir}. Does {experiment_name} match the minknow experiment name and is minknow already running long enough?",
-            )
+    if not minknow.is_fastq_dir(fastq_dir) and experiment_name not in fastq_dir:
```

Comment:
I am not completely clear on this conditional, but maybe I'm dumb

Either we get:
(a) /var/lib/minknow/data/<expt_name>/*/*/fastq_pass
(b) /var/lib/minknow/data
(c) some erroneous path

If (a), we get False and False
If (b), we get True and True
If (c), we will evaluate to True in most normal cases, unless they get the path order wrong (e.g. fastq_pass/<expt_name>)

How are we checking they didn't make a mistake with the fastq_dir? In the click option? Why do we need both checks here?

Reply:
Removed the second check